changelogs/unreleased/5657-shadialtarsha-minor.md

```diff
@@ -0,0 +1,36 @@
+## Disable the virtualhost's Global RateLimit policy 
+
+Setting `global.disabled` flag to false on a specific route should disable the vhost global rate limit policy.
+
+### Sample Configurations
+#### httpproxy.yaml
+```yaml
+apiVersion: projectcontour.io/v1
+kind: HTTPProxy
+metadata:
+  name: echo
+spec:
+  virtualhost:
+    fqdn: local.projectcontour.io
+    rateLimitPolicy:
+      global:
+        disabled: true
```

Comment:
Seems like a better example of this fix would be having a non-disabled virtualhost-level global rate limit policy (rather than having `disabled: true` here), and then disabling it for a specific route like you have below (unless I'm misunderstanding the issue here).

Reply:
Thanks for the suggestion!